build.gradle

```diff
@@ -158,18 +177,17 @@ subprojects {
 
   artifacts {
     archives srcJar
+    archives testJar
+    archives testSrcJar
     archives javadocJar
   }
 
   plugins.withType(ScalaPlugin) {
-    //source jar should also contain scala source:
-    srcJar.from sourceSets.main.scala
-
     task scaladocJar(type:Jar) {
       classifier = 'scaladoc'
       from '../LICENSE'
       from '../NOTICE'
-      from scaladoc
+      from scaladoc.destinationDir
```

Comment:
This was to be consistent with the javadocJar task. I don't care which we use, just wanted them to be the same.